src/lib/isMobilePhone.js

```diff
@@ -159,6 +159,7 @@ const phones = {
   'ar-YE': /^(((\+|00)9677|0?7)[0137]\d{7}|((\+|00)967|0)[1-7]\d{6})$/,
   'ar-EH': /^(\+?212|0)[\s\-]?(5288|5289)[\s\-]?\d{5}$/,
   'fa-AF': /^(\+93|0)?(2{1}[0-8]{1}|[3-5]{1}[0-4]{1})(\d{7})$/,
+  'sn-SN': /^(221|00221|\+221)?(77|78|75|70|76)[0-9]{7}$/,
```

Comment:
```suggestion
  'sn-SN': /^(\+?221|0)?(70|72|7[5-9])[0-9]{7}$/,
```
According to the most recent document in https://www.itu.int/oth/T02020000B8/en , this should be the regular expression. I have also updated the first part to be more in line with other locales (even though there are major inconsistencies)
Tests should also be updated for this

Please also move this to it's proper alphabetical location, like in the README.